xep-0440.xml

```diff
@@ -71,35 +79,35 @@
 
 <section1 topic='Announcing the SASL Channel-Binding Type Capability' anchor='sasl-cb-type'>
 
-  <p>This protocol consists of a single optional extension element
-  named 'sasl-channel-binding' qualified by the 'urn:xmpp:sasl-cb:0'
-  namespace. The 'sasl-channel-binding' element MUST contain one or
+  <p>This protocol consists of a stream feature named 'sasl-channel-binding'
+  qualified by the 'urn:xmpp:sasl-cb:0' namespace.
+  The 'sasl-channel-binding' element MUST contain one or
   more 'channel-binding' elements, of which each MUST have an
   attribute with the name 'type'. The value of the 'type' attribute
   SHOULD be the "Channel-binding unique prefix" of a channel-binding
   type which was registered with the &iana-cb-types;.</p>
 
   <p>A server declares that it supports particular channel-binding
   types by listing the supported types via the 'sasl-channel-binding'
-  element defined herein. The 'sasl-channel-binding' element could
-  appear as child element to the SASL &lt;mechanisms/&gt;
+  stream-feature defined herein. The 'sasl-channel-binding' element could
```

Comment:
Sorry, here is another one that is not a compound adjective, and hence should drop the hyphen.

Reply:
Updated